b2share/modules/schemas/serializers.py

```diff
@@ -110,3 +137,89 @@ def community_schema_to_json_serializer(community_schema, code=200,
     if headers is not None:
         response.headers.extend(headers)
     return response
+
+
+def new_schema_to_json_serializer(schema, code=200, headers=None):
+    """Serializes schema to response.
```

Comment:
=>`Serialize schema to response.` (without "*s*"). Same for other docstrings.
```
The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".
```
https://www.python.org/dev/peps/pep-0257/#specification